nptyping/assert_isinstance.py

```diff
@@ -51,5 +51,5 @@ def assert_isinstance(
     if message:
         assert isinstance(instance, cls), message
     else:
-        assert isinstance(instance, cls)
+        assert isinstance(instance, cls), f"instance={instance!r}, cls={cls!r}"
```

Comment:
The function can now be more concise:
```python
    ...
    message = message or f"instance={instance!r}, cls={cls!r}"
    assert isinstance(instance, cls), message
    return True
```

Reply:
Hi @ramonhagenaars, thanks for the feedback! I've updated the PR in commit https://github.com/ramonhagenaars/nptyping/pull/66/commits/609a6389f814d6c59ddeadd086514f2026267506.